docker-compose.yml

```diff
@@ -13,6 +13,13 @@ services:
               source: ${TESTSET_DATA_FOLDER}/reference_genomes.json
               target: /workspace/reference_genomes.json
               read_only: true
+        healthcheck:
+          test: ["CMD-SHELL",
+            "wget -qO- http://localhost:8080/sample/aggregated | grep -o '\"count\":[0-9]*' | sed 's/[^0-9]//g' | xargs test 0 -ne || exit 1"]
```

Comment:
This indicates whether LAPIS __and__ SILO are healthy. IMO it should reflect LAPIS' state only.

Anyway, it would actually better to implement that in LAPIS itself.

Reply:
Using some sort of info endpoint. Spring provides health endpoints, they simply need to be enabled. Let's do https://github.com/GenSpectrum/LAPIS/issues/813 instead.

Reply:
I added #32 but do not want this blocking the usability for now